app/views/course/assessment/assessments/show.html.slim

```diff
@@ -34,6 +34,22 @@
       tr.end_at
         th = t('.end_at')
         td = format_datetime(@assessment.end_at) if @assessment.end_at.present?
+      - if can?(:manage, @assessment)
+        - if @assessment.autograded?
+          tr.skippable
+            th = t('.skippable')
+            td = @assessment.skippable ? t('.allow') : t('.disallow')
+          tr.allow_partial_submission
+            th = t('.allow_partial_submission')
+            td = @assessment.allow_partial_submission ? t('.allow') : t('.disallow')
+        tr.graded_test_case_types
+          th = t('.graded_test_case_types')
+          - used_test_types = []
+          - used_test_types.tap do |test_type|
```

Comment:
Probably clearer to just do `used_test_types.push(...) if ...` for these 3 lines.

In the `graded_test_case_types` function, `tap` allowed us to immediately return the correct value without have to init a variable at the start then write it at the end of the function to return it. This benefit doesn't apply here.